mevislab.github.io/content/tutorials/visualization/visualizationexample9.md

```diff
@@ -19,24 +19,24 @@ In this tutorial, our focus will be on using the `AnimationRecorder` module to g
 
 ## Steps to do
 
-Open the network of Example 6.2: Visualisation using `PathTracer`, add a `SoSeparator` and an `AnimationRecorder` into your workspace and connect them as shown down below. 
+Open the network of *Example 6.2: Visualisation using `PathTracer`*, add a `SoSeparator` and an `AnimationRecorder` into your workspace and connect them as shown down below. 
 
 `SoSeparator`’s role is to isolate and organize scene components, enabling precise control over the animation's focus without affecting the entire scene. 
 
 `AnimationRecorder`, equipped with a key frame editor, facilitates smooth transitions between different states of the 3D scene by allowing field additions via drag-and-drop. The tree view allows for easy reordering of fields and modules, enhancing overall organization.
 
 ![ AnimationRecorder](/images/tutorials/visualization//pathtracer/Example9_1.png " AnimationRecorder")
 
-Define the following LUTs in `SoLUTEditor` of the knee or load this [XML file](/examples/visualization/example6/LUT_AnimationRecorder.xml) to use a pre-defined LUT.
+Define the following LUTs in `SoLUTEditor` of the knee or load this [XML file](/examples/visualization/example6/LUT_AnimationRecorder1.xml) with `LUTLoad1` to use a pre-defined LUT.
 
-![ SoLUTEditor](/images/tutorials/visualization//pathtracer/V9_2.png " SoLUTEditor")
+![ SoLUTEditor](/images/tutorials/visualization//pathtracer/V9_LUT.png " SoLUTEditor")
 
 
 Open the `AnimationRecorder` module and click on *New* to initiate a new animation, selecting a filename for the recorded video.
 
 At the bottom of the `AnimationRecorder` panel, you'll find the key frame editor, which is initially enabled. It contains the camera track with a key frame at position *0*. The key frame editor at the bottom serves as a control hub for playback and recording.
 
-![ AnimationRecorder](/images/tutorials/visualization//pathtracer/Example9_4.png " AnimationRecorder")
+![ AnimationRecorder](/images/tutorials/visualization//pathtracer/V9_AnimationRecorder.png " AnimationRecorder")
 
 Open the `AnimationRecorder` panel and locate the key frame editor. Move to the desired timepoint on the timeline, adjusting the camera to the preferred position in your 3D scene. Upon double-clicking on the timeline at the chosen timepoint, a Key frame will be added and a window labeled *Edit Camera Key Frame* will open.
 
```

Comment:
Additional Info: Close the SoExaminerViewer while using the AnimationRecorder to prevent duplicate renderings and save resources.